frontend/packages/console-app/src/components/modals/restore-pvc/restore-pvc-modal.tsx

```diff
@@ -185,6 +194,7 @@ const RestorePVCModal = withHandlePromise<RestorePVCModalProps>(
                   filter={onlyPvcSCs}
                   id="restore-storage-class"
                   required
+                  selectedKey={volumeSnapshotAnnotations?.[snapshotPVCStorageClassAnnotation]}
```

Comment:
Better create function to do this.
```suggestion
                  selectedKey={volumeSnapshotAnnotations?.[snapshotPVCStorageClassAnnotation] || onlyPvcSCs?.[0]}
```

Reply:
could you elaborate on this ??? `onlyPvcSCs` is a boolean value and the output of this would be undefined. i don't see why we need to pass undefined to selectedKey.  The first expression would itself evaluate to undefined if the annotation is not there.